types/index.d.ts

```diff
@@ -2309,6 +2309,10 @@ interface MsSqlConnectionConfigBase {
     enumName: string;
   }
 
+  interface DefaultToOptions {
+    constraintName?: string;
```

Comment:
adding comment explaining that this only works for mssql might be helpful

Reply:
done